### DISCUSSION TITLE
[Feature Idea] Atom Visualizer Feature

### DISCUSSION THREAD

**Discussion by tori-obregon** [jotaijs/jotai-devtools#146] [Ideas]
My friends and I have built a new tab for the devtool called Atom Graph where devs can view their application's state. The primary benefit is too see all the atomized state as nodes within a ReactFlow container, and view the relationships between the pieces of state. Perhaps it can be included in a future iteration of the devtool.

Here is the repo (default branch - mainApril): https://github.com/oslabs-beta/jotai-devtools/tree/mainApril 

To preview:
- npm install
- npm run compile
- npm run build-storybook
- npm run storybook

![Screenshot 2024-05-21 at 10 12 01 AM](https://github.com/jotaijs/jotai-devtools/assets/155405210/7934b1ea-5743-4e7b-9057-29c7eb9cec2d)
![Screenshot 2024-05-21 at 12 15 51 PM](https://github.com/jotaijs/jotai-devtools/assets/155405210/0270cc2b-f568-473c-b753-f2679a0d9ce6)

**Comment by WebDevMcFarland** [comment]
https://github.com/jotaijs/jotai-devtools/assets/149956718/5ff3e6c7-a9f2-4601-b59f-19646fdc3e5f

**Comment by arjunvegda** [comment]
This is great! Thanks for working on this. I'll think more about how we could provision this (perhaps via extensions?) in the future and let you know.

**Reply by WebDevMcFarland** [reply]
Sounds good, thanks Arjun!
